waku/waku_rln_relay/constants.nim

```diff
@@ -58,7 +58,7 @@ const
 # rln-v2: rate commitments are used for the Merkle tree construction, defaulting the UserMessageLimit to 20
 # the root is created locally, using createMembershipList proc from waku_rln_relay_utils module, and the result is hardcoded in here
 when defined(rln_v2):
-  const StaticGroupMerkleRoot* = "0a15ba7c5753ee78e8126603113028a343c1a01ffcb389565c76626be158e964"
+  const StaticGroupMerkleRoot* = "2c149e48886b5ba3da2edf8db8d7a364ae7a25618489c04cf0c0380f7cdd4d6f"
```

Comment:
Why this needs to be changed if the `StaticGroupKeys` seems not to be changed?

Reply:
This is because the leaves in the tree are derived from `StaticGroupKeys` in rln-v2, the leaf inserted is actually `poseidon([identity_commitment, user_message_limit])` for the static group we default to 20 as the `user_message_limit` :)